chain/x/zoracle/internal/keeper/raw_request.go

```diff
@@ -62,6 +63,10 @@ func (k Keeper) AddNewRawDataRequest(
 		return types.ErrRequestNotFound(types.DefaultCodespace)
 	}
 
+	ctx.GasMeter().ConsumeGas(
+		k.GasPerRawDataRequest(ctx)*uint64(len(request.RequestedValidators)),
```

Comment:
Is it possible to make it overflow here? no?

Reply:
It has a chance if the length of the validators set is too long. We loop consume gas per validator and check overflow in gas consume(cosmos work)?

Reply:
Let's leave it like this for now. I think the chance is minimal.